systemtests/src/main/java/io/enmasse/systemtest/Environment.java

```diff
@@ -95,11 +96,15 @@ private Environment() {
         if (installType == EnmasseInstallType.OLM) {
             log.info(debugFormat, OLM_INSTALL_TYPE, olmInstallType.name());
         }
-        if (url.startsWith("https://api")) {
-            try {
-                this.kubernetesDomain = new URL(url).getHost().replace("api", "apps");
-            } catch (Exception ex) {
-                throw new RuntimeException(ex);
+        if (Strings.isNullOrEmpty(this.kubernetesDomain)) {
+            if (url.startsWith("https://api")) {
```

Comment:
I don't understand the reason for checking "https://api" specifically. Could you add a comment for it so that it is clearer?

Reply:
@lulf it is because ocp4 clusters. yup I can add it